packages/core/src/extensions/Hyperlinks/menus/helpers/PanelTextInput.tsx

```diff
@@ -30,6 +29,7 @@ export interface State {
   value?: string;
 }
 
+const isMacOS = () => /Mac/.test(navigator.platform);
```

Comment:
Let's move this to a "util" file. We don't want this logic spread around the codebase